tools/snitch/snitcher/src/lib.rs

```diff
@@ -451,67 +413,3 @@ where
         })
     })
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
why all these tests are removed?

Reply:
They need to be re-written I think. They are more like end-to-end tests and not unit tests. I can add them back in if you like but in order to run them one has to setup up a complete environment to run successfully.